ecommerce/colaraz_features/api/views.py

```diff
@@ -0,0 +1,34 @@
+"""
+Views for colaraz API.
+"""
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from ecommerce.extensions.api.permissions import IsStaffOrOwner
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import serializers as rest_serializers
+
+from ecommerce.colaraz_features.api import serializers
+
+class SiteOrgViewSet(viewsets.ViewSet):
+    """
+    View set to enable creation of site, organization and theme via REST API.
+    """
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (IsAuthenticated, IsStaffOrOwner,)
+    serializer_class = serializers.SiteOrgSerializer
+
+    def create(self, request):
+        """
+        Perform creation operation for site, organization, site theme and site configuration.
+        """
+        try:
+            serializer = self.serializer_class(data=request.data)
+            if serializer.is_valid(raise_exception=True):
+                serializer.create(serializer.validated_data)
+        except (rest_serializers.ValidationError, ValueError, NameError) as ex:
+            return Response(str(ex), status=status.HTTP_400_BAD_REQUEST)
+        except:
+            return Response("Request data is unappropriate", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
```suggestion
            return Response('Request data is unappropriate', status=status.HTTP_400_BAD_REQUEST)
```

Reply:
Shouldn't we send JSON response since its an API view?

Reply:
```
from django.http import JsonResponse

...

return JsonResponse({'error': 'Request data is unappropriate'}, status=status.HTTP_400_BAD_REQUEST)
```

Reply:
Updated.